### ISSUE TITLE
Stops reversing all together, not just after stopping.

### ISSUE THREAD

**Issue by CanadaFarmer** [Flohhhhh/no-auto-reverse#1]
Installed this in our server to test it out, but it just stops players from being able to reverse at all...

**Comment by Flohhhhh** [comment]
Yeah this doesn't work. Haven't finished it yet. Sorry about that I sound make it private haha.
